src/universal/modules/notifications/components/ProjectInvolves.js

```diff
@@ -48,48 +50,63 @@ const ProjectInvolves = (props) => {
   const action = involvementWord[involvement];
   const contentState = convertFromRaw(JSON.parse(content));
   const editorState = EditorState.createWithContent(contentState, editorDecorators);
+  const projectStyles = css(
+    styles.projectListView,
+    styles[status],
+    tags.includes('private') && styles.private
+  );
   return (
-    <Row>
+    <Row compact>
       <div className={css(styles.icon)}>
-        <IconAvatar icon="check-square" size="medium" />
+        <IconAvatar icon={involvement === MENTIONEE ? 'at' : 'id-card-o'} size="small" />
       </div>
       <div className={css(styles.message)}>
         <div className={css(styles.messageText)}>
-          <span className={css(styles.messageVar)}>{changeAuthorName}</span>
-          <span>{' has'}</span>
-          <span className={css(styles.messageVar)}> {` ${action} you`} </span>
+          <b>{changeAuthorName}</b>
+          <span>{' has '}</span>
+          <b><i>{`${action} you`}</i></b>
           {involvement === MENTIONEE ? ' in' : ''}
-          <span>{' a project for'}</span>
-          <span className={css(styles.messageVar)}> {teamName} </span>
-          <span>{'.'}</span>
+          <span>{' a project for '}</span>
+          <span className={css(styles.messageVar, styles.notifLink)} onClick={gotoBoard} title={`Go to ${teamName}’s Board`}>
+            {teamName}
+          </span>
+          <span>{':'}</span>
         </div>
-        <div className={css(styles.projectListView)}>
+        <div className={projectStyles}>
           <Editor
             readOnly
             editorState={editorState}
           />
+          <div className={css(styles.owner)}>
```

Comment:
so this isn't blocked by the `projects-to-relay` branch, let's wrap this in a `{teamMember && ...}` block so it's OK if the teamMember is null